test/phoenix_html/form_test.exs

```diff
@@ -449,19 +450,26 @@ defmodule Phoenix.HTML.FormTest do
 
     assert safe_to_string(
              time_input(:search, :key, value: ~T[23:00:07.001], id: "key", name: "search[key][]")
-           ) == ~s(<input id="key" name="search[key][]" type="time" value="23:00">)
+           ) == ~s(<input id="key" name="search[key][]" type="time" value="23:00:07.001">)
```

Comment:
This is changing the behaviour, no? I think our default precision should be `:minute` and we handle it especially.